### PR TITLE
need API access to language manager settings #3479

### PR DESCRIPTION
**Pull request by mnavasiolava**

### CODE REVIEW

**Comment by scottdraves** [comment]
ok this partly works, but if you try to use groovy only, it does not work.
see this notebook:
https://pub.beakernotebook.com/#/publications/56ba327b-9b29-4029-8152-535674e9c91b
if you run the 1st cell it fails with the visible error.
if you run the JS cell, then run the Groovy cell, then it works.
if you update the classpath, then evaluate the groovy cell, you get the old value, you need to run the JS to update it.

**Comment by scottdraves** [comment]
also, see the 3rd cell, beaker object should not have this be visible, it should be removed.

File: core/src/main/web/app/helpers/helper.js [hunk 1]
```diff
@@ -100,6 +100,41 @@
...
+        if (beaker) {
+          beaker.language = {};
+        }
+      },
+      setLanguageManagerSettingsToBeakerObject: function (plugin) {
+        var beakerObject = this.getBeakerObject().beakerObj;
+        if (beakerObject && beakerObject.language) {
+          var spec = plugin.spec;
+          var beakerLanguageSettings = {};
+          for (var property in spec){
```
Comment (tomca32) [inline core/src/main/web/app/helpers/helper.js]: This is fine but you might want to use lo-dash sugar for iterating own properties:
https://lodash.com/docs#forOwn

I just think it cuts down on the boilerplate, but its your choice

Comment (mnavasiolava) [inline core/src/main/web/app/helpers/helper.js]: good idea. thanks @tomca32

**Comment by scottdraves** [comment]
:+1:

**Pull request merged**